deployment-detector/README.md

```diff
@@ -1,27 +1,26 @@
-# Challenge-1
+# Nethermind Forta Bots Deployer Detector
 
 ## Description
+This  monitors bot deployments from the Nethermind deployer address (0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8)
 
-This agent detects forta bots that are deployed on behalf of nethermind.
-
-## Supported Chains
-
+## Supported ChainsS
 - Polygon
 
-
 ## Alerts
-
-Describe each of the type of alerts fired by this agent
-
-- FORTA-1
-  - Fired when Nethermind contract at :"0x88dc3a2284fa62e0027d6d6b1fcfdd2141a143b8" deploys an agent
+- NETH-BOT-DEPLOYER
+S
+  - Fired when Nethermind contract at: [0x88dc3a2284fa62e0027d6d6b1fcfdd2141a143b8](https://polygonscan.com/tx/0x88dc3a2284fa62e0027d6d6b1fcfdd2141a143b8) deploys a bot
   - Severity is always set to "info" 
   - Type is always set to "info" 
   - Metadata fields:
-     - agentId: agentId of the deployed agent
-     - metadata: ipfs of the metadat of the agent
-     - chainsIds: list of networks supported 
 
-## Test Data
+     - `agentId`: NETH-BOT-DEPLOYER,
```

Comment:
`NETH-BOT-DEPLOYER` is the alertId, not the botId. The botId should be a long string of characters that you can get from the event arguments.